install/kubernetes/cilium/values.yaml

```
@@ -1167,7 +1167,7 @@ operator:
 
   # -- Additional cilium-etcd-operator container arguments
```

I think we should update this comment to:
```suggestion
  # -- Additional cilium-operator container arguments
```